package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "openapi-zod-client-monorepo",
     "private": true,
-    "version": "1.4.18",
+    "version": "1.4.19",
```

Comment:
no need to update that manually, changeset is going to take care of it when merged 🙏